matplotcheck/tests/test_notebook.py

```diff
@@ -0,0 +1,145 @@
+"""Tests for the notebook module"""
+
+import pytest
+import matplotlib.pyplot as plt
+from matplotcheck.base import PlotTester
+import matplotcheck.notebook as nb
+
+
+@pytest.fixture
+def locals_dictionary_good():
+    """Abriged locals dictionary of a jupyter notebook with import in the proper
+    place"""
+    return {
+        "__name__": "__main__",
+        "__doc__": "Automatically created module for IPython interactive",
+        "_ih": ["", "import os", 'print("Good notebook")', "locals()"],
+        "In": ["", "import os", 'print("Good notebook")', "locals()"],
+        "Out": {},
+        "_i": 'print("Good notebook")',
+        "_ii": "import os",
+        "_iii": "",
+        "_i1": "import os",
+        "_i2": 'print("Good notebook")',
+        "_i3": "locals()",
+    }
+
+
+@pytest.fixture
+def locals_dictionary_bad():
+    """Abriged locals dictionary of a jupyter notebook with import in the wrong
+    place"""
+    return {
+        "__name__": "__main__",
+        "__doc__": "Automatically created module for IPython interactive",
+        "_ih": ["", 'print("Bad notebook")', "import os", "locals()"],
+        "In": ["", 'print("Bad notebook")', "import os", "locals()"],
+        "Out": {},
+        "_i": "import os",
+        "_ii": 'print("Bad notebook")',
+        "_iii": "",
+        "_i1": 'print("Bad notebook")',
+        "_i2": "import os",
+        "_i3": "locals()",
+    }
+
+
+def test_notebook_convert_single_axes(basic_polygon_gdf):
+    """Test convert single axes."""
+    fig, ax = plt.subplots()
+    basic_polygon_gdf.plot(ax=ax)
+    store = nb.convert_axes(plt)
+    with pytest.raises(AssertionError, match="Axis lines are displayed on"):
+        PlotTester(store).assert_axis_off()
+    plt.close()
+
+
+def test_notebook_convert_multi_axes(basic_polygon_gdf):
+    """Test convert multi axes."""
+    fig, (ax1, ax2) = plt.subplots(1, 2)
+    basic_polygon_gdf.plot(ax=ax1)
+    basic_polygon_gdf.plot(ax=ax2)
+    store = nb.convert_axes(plt, which_axes="all")
+    for axes in store:
+        with pytest.raises(
+            AssertionError, match="Axis lines are displayed on"
+        ):
+            PlotTester(axes).assert_axis_off()
+    plt.close()
+
+
+def test_notebook_convert_last_axes(basic_polygon_gdf):
+    """Test convert last axes."""
+    fig, (ax1, ax2) = plt.subplots(1, 2)
+    basic_polygon_gdf.plot(ax=ax2)
+    store = nb.convert_axes(plt, which_axes="last")
+    with pytest.raises(AssertionError, match="Axis lines are displayed on"):
```

Comment:
this test doesn't fully make sense to me. it seems like you should have a plot with two subplots. plot something on both and then check that maybe the title of the second plot is correct. so add titles to both and then make sure  you test the title in the second plot. i the title matches the title of ax2 it is working as we think it should.

It might be better to test this way to ensure we are getting the axis object that we think we are. 

id suggest this for all tests unless you just want to ensure that an axis was returned.